deployments/gnosis.diamond.json

```
@@ -97,13 +97,21 @@
         "Name": "GasZipFacet",
         "Version": "2.0.0"
       },
+      "0x5656E307E4dEd573695Dd3eAC64EC4778F910DcA": {
```

Biome (1.9.4)

[error] The key 0x5656E307E4dEd573695Dd3eAC64EC4778F910DcA was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)